corehq/apps/hqwebapp/templates/hqwebapp/includes/inactivity_modal_data.html

```diff
@@ -5,4 +5,9 @@
 {% registerurl 'domain_login_new_window' %}
 {% registerurl 'ping_login' %}
 {% registerurl 'ping_session' %}
-{% include 'hqwebapp/includes/bootstrap3/inactivity_modal.html' %}
\ No newline at end of file
+
+{% if use_bootstrap5 %}
```

Comment:
@biyeun FYI I think the migration tool should have picked this up (and tried to split this file), but I realize you split most of hqwebapp a while ago and the tool has been updated significantly since then.